src/main/resources/assets/jei/lang/tr_tr.json

```diff
@@ -118,4 +136,7 @@
   "description.jei.wooden.door.1": "Wooden Doors allow you to block monsters from entering your building.\\nTesting sentences.",
   "description.jei.wooden.door.2": "Clicking on a door changes its state from open to closed and vice versa.",
   "description.jei.wooden.door.3": "Wooden Doors can be opened/closed via Redstone Circuits."
```

Comment:
Besides still having several untranslated lines above, you forgot a comma at the end of this line :)